{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/faqs/templates/faqs/faq_detail.html

```diff
@@ -1,5 +1,34 @@
 {% extends "base.html" %}
 
+{% block schema %}
+  <script type="application/ld+json">
+    {
+      "@context": "http://schema.org",
+      "@type": "Question",
+      "name": "{{ faq.title }}",
+      "text": "{{ faq.question }}",
+      "dateCreated": "{{ faq.date_created }}",
+      "datePublished”: "{{ faq.date_created }}",
+      "author": {
+        "@type": "Organization",
+        "email": "{{ settings.SERVER_EMAIL }}",
+        "name": "{{ settings.SITE_NAME }}",
+      }
```

Comment:
broken JSON - missing comma. This is why I think it would be better to JSON-encode it properly (see my comments on the other template) to make these errors impossible.